test/createAzureResource.test.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import { ResourceManagementClient } from 'azure-arm-resource';
+// tslint:disable-next-line:no-require-imports
+import WebSiteManagementClient = require('azure-arm-website');
+import { Site } from 'azure-arm-website/lib/models';
+import { IHookCallbackContext, ISuiteCallbackContext } from 'mocha';
+import * as vscode from 'vscode';
+import { AzureTreeDataProvider, DialogResponses, TestAzureAccount, TestUserInput } from 'vscode-azureextensionui';
+import { ext } from '../src/extensionVariables';
+import { localize } from '../src/localize';
+import { FunctionAppProvider } from '../src/tree/FunctionAppProvider';
+import * as fsUtil from '../src/utils/fs';
+
+// This ensures that this will only run on nightly cron builds
+if (process.env.TRAVIS_EVENT_TYPE === 'cron') {
```

Comment:
Much better would be to either:

1) create longRunningTest()/longRunningTest() functions that wrap suite()/test() and do the skip automatically

longRunningSuite("Create Azure Resources", () => {

2) use JavaScript decorators to indicate long-running suites/tests (I've never tried this in JavaScript, but you can find info here: https://github.com/tc39/proposal-decorators)

@longRunning
suite("Create Azure Resources", () => ...

Reply:
Example of 1: https://github.com/Microsoft/vscode-docker/blob/145935f31ac58f2e409db23214e8280cde67845c/test/buildAndRun.test.ts#L115